app/web/features/profile/edit/EditProfile.tsx

```diff
@@ -155,6 +156,20 @@ export default function EditProfileForm() {
       )}
       {user ? (
         <>
+          <div className={classes.helpTextContainer}>
+            <Typography variant="body1">
+              <Trans i18nKey="profile:edit_profile_helper_text">
+                Looking for some inspiration on where to start?{" "}
+                <StyledLink
+                  variant="body1"
+                  href="https://help.couchers.org/hc/couchersorg-help-center/articles/1715658357-how-to-write-a-request-that-gets-accepted"
```

Comment:
```suggestion
                  href="https://help.couchers.org/hc/couchersorg-help-center/articles/1725919197-how-do-i-create-a-great-profile"
```
This had the wrong hyperlink

Reply:
Thanks for noticing! Fixed!